src/commands/platforms/destroy.ts

```diff
@@ -7,8 +7,8 @@ import { AccountUtils } from '../../common/utils/account';
 import { PlatformUtils } from '../../common/utils/platform';
 
 export default class PlatformDestroy extends Command {
-  static aliases = ['platform:destroy', 'platforms:destroy'];
-  static description = 'Destroy a platform';
+  static aliases = ['platforms:deregister', 'platform:destroy', 'platforms:destroy'];
+  static description = 'Deregister a platform from Architect';
```

Comment:
@davidthor , @tjhiggins : proposing these changes to align the language of the inverse command ("Register a new platform...")

Plus I think it's scary to say we're "destroying" a platform that they brought.

Reply:
I'm not opposed on the condition we are consistent and update to environment:deregister too. Wait for David to respond before updating.

Reply:
I'm cool with it since we aren't actually changing anything in the environment when we remove it.

Reply:
Do we no longer remove all resources in the environment when we do an environment:destroy?

I liked environment:destroy because an Environment is an Architect construct with architect-created resources, whereas the platform felt like something that they were bringing to the table and we weren't destroying it, just deregistering it.

Reply:
Ah right - that is a confusing disconnect from the UI where removing "deregisters".

Reply:
Ha wait - `architect destroy` handles the destruction of actual cloud resources. `architect environment:destroy` just deregisters the same as platform.